docs/spec/font.en-US.md

```diff
@@ -22,16 +22,15 @@ While defining the font system for a visual system, we propose to start from the
 In order to implement a good font system, the first thing is to choose an appropriate font family. Ant Design prefers the system default font family and then also provides a set of alternative font libraries to maintain readability for screens on different platforms and browsers and to make sure it's always user friendly, stable and professional to end user.
 
 ```css
-font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
-             "Helvetica Neue", Helvetica, "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei",
-             SimSun, sans-serif;
+font-family: "Chinese Quote", -apple-system, BlinkMacSystemFont, "Segoe UI", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial, sans-serif,
```

Comment:
I removed `SimSun` here because [it wasn't actually used](https://github.com/ant-design/ant-design/pull/11567/files#diff-4b7c05557620faa918b0919c047d049a).

I didn't change the `font.zh-CN.md` file, but the PR is configure to "Allow edits from maintainers.". Feel free to edit the Chinese page and to tweak the line breaking if you want shorter lines in the docs.

Reply:
Hi @tibdex the adding of "Chinese Quote" replaced Segoe UI everywhere in IE11  :( 
Had to fix it manually

![2018-08-06_113446](https://user-images.githubusercontent.com/29225198/43709525-61401bb6-996d-11e8-8a2b-8812ad95fe3d.png)

(chrome looks ok)

Reply:
Hi @Gyokur0, as you can see in my comment above, I only added `"Chinese Quote"` in the docs to keep them in sync with the actual font stack used in Less files.

This is the font stack before my commit:

https://github.com/ant-design/ant-design/blob/1d924300e7b682a6b8c7d723ad0a0a7721895f7d/components/style/themes/default.less#L37-L39

Note the presence of `"Chinese Quote"`.

And after:

https://github.com/ant-design/ant-design/blob/b6e9646a6ad5fef497c48d955cc0e6141e6146f2/components/style/themes/default.less#L37-L39

Still starts with `"Chinese Quote", -apple-system, BlinkMacSystemFont, "Segoe UI"`.

I don't see how what you described would have happened.

Besides, I just visited https://ant.design/components/table/, which ships with this commit, on IE 11 and it displays with Segoe UI as expected.